app/src/main/res/values-en/strings.xml

```diff
@@ -783,7 +783,7 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_buildingType_abandoned">Abandoned</string>
     <string name="quest_buildingType_abandoned_description">unused building whose original function or purpose is unknown</string>
     <string name="quest_buildingType_ruins">Ruins</string>
-    <string name="quest_buildingType_ruins_description">derelict, ruined and unidentifiable building</string>
+    <string name="quest_buildingType_ruins_description">derelict, ruined and/or unidentifiable building</string>
```

Comment:
It shouldn't be used for an unidentifiable building which isn't a ruin/derelict.

Reply:
Unidentifiable buildings that are derelict or ruined?

Reply:
I.e. they are unidentifiable **because** they are derelict or ruined.

Reply:
Okay, this one sounds hard so I just put it back.